scripts/www/build-cfg-help.pl

```diff
@@ -267,8 +270,28 @@ ()
         $data->{'default_doc'} = "";
         $data->{'default_if_none'} = "";
 
-        print "DEBUG: new option: $name\n" if $verbose;
+        print "DEBUG: line $linecount: new option: $name\n" if $verbose;
         next;
+    } elsif ($_ =~ /^IF (.*)$/) {
+        my $cond = $1;
+        push(@ifelse, $1);
```

Comment:
I would push the current line number so that warnings about unclosed `IF`s (see another change request) can tell the user where the broken `IF` started.

```suggestion
        push(@ifelse, "$.: $1");
```